packages/server-renderer/src/render.ts

```diff
@@ -77,22 +77,23 @@ function waitForServerPrefetch(vm, resolve, reject) {
 }
 
 function renderNode(node, isRoot, context) {
-  if (node.isString) {
+  const {asyncFactory, componentOptions, isComment, isString, raw, text, tag } = node
```

Comment:
All those variables are only used once. Destructuring them makes the code harder to read or maintain, because we would need to change 2 lines for 1 logical change.